site/docs/configuration/reference.md

```diff
@@ -35,6 +35,9 @@ Here is the main structure of the promptfoo configuration file:
 | evaluateOptions.repeat          | number                                                                                           | No       | Number of times to run each test case . Defaults to 1                                                                                                                                                       |
 | evaluateOptions.delay           | number                                                                                           | No       | Force the test runner to wait after each API call (milliseconds)                                                                                                                                            |
 | evaluateOptions.showProgressBar | boolean                                                                                          | No       | Whether to display the progress bar                                                                                                                                                                         |
+| evaluateOptions.cache           | boolean                                                                                          | No       | Whether to use disk cache for results (default: true)                                                                                                                                                       |
+| evaluateOptions.timeoutMs       | number                                                                                           | No       | Timeout in milliseconds for each individual test case/provider API call. When reached, that specific test is marked as an error. Default is 0 (no timeout).                                                 |
```

Comment:
@mldangelo, `timeoutMs` and `maxEalTimeMs` are both valid `evaluateOptions` that were not included in the docs and are not exposed in the eval CLI options.  Want to check whether there is a good reason for this and, if so, whether I should revert this change.